sklearn/linear_model/sag.py

```diff
@@ -170,6 +170,8 @@ def sag_solver(X, y, sample_weight=None, loss='log', alpha=1.,
     Ridge, SGDRegressor, ElasticNet, Lasso, SVR, and
     LogisticRegression, SGDClassifier, LinearSVC, Perceptron
     """
+    if not warm_start_mem:
+        warm_start_mem = dict()
```

Comment:
Consistency is important indeed. Fixed.

2015-10-31 13:09 GMT+01:00 Joel Nothman <notifications@github.com>:

> In sklearn/linear_model/sag.py
> <https://github.com/scikit-learn/scikit-learn/pull/5648#discussion_r43571179>
> :
>
> > @@ -170,6 +170,8 @@ def sag_solver(X, y, sample_weight=None, loss='log', alpha=1.,
> >      Ridge, SGDRegressor, ElasticNet, Lasso, SVR, and
> >      LogisticRegression, SGDClassifier, LinearSVC, Perceptron
> >      """
> > +    if not warm_start_mem:
> > +        warm_start_mem = dict()
>
> There's no difference, effectively, but because you insisted on it
> elsewhere, I thought it strange that you should be inconsistent.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/scikit-learn/scikit-learn/pull/5648/files#r43571179>.
>